interlok-core/src/test/java/com/adaptris/core/runtime/MessageDigestErrorEntryTest.java

```diff
@@ -120,9 +116,10 @@ public void testToString() throws Exception {
   }
 
   private File deleteLater(Object marker) throws IOException {
-    File f = File.createTempFile(this.getClass().getSimpleName(), "");
-    f.deleteOnExit();
-    cleaner.track(f, marker, FileDeleteStrategy.FORCE);
-    return f;
+    File file = File.createTempFile(this.getClass().getSimpleName(), "");
+    file.deleteOnExit();
```

Comment:
It probably doesn't matter but `deleteOnExit` actually grows a list of strings inside the JVM (because it needs to keep a track of all the files it should delete on JVM exit). It's test code, so it's neither here nor there, but since we're already doing `cleaner.track()` it isn't necessary.